test/mocha/40-database.js

```diff
@@ -60,8 +61,10 @@ describe('Keystores Database Tests', () => {
       executionStats.totalDocsExamined.should.equal(1);
       executionStats.executionStages.inputStage.inputStage.stage
         .should.equal('IXSCAN');
+      executionStats.executionStages.inputStage.inputStage.keyPattern
+        .should.eql({'config.id': 1});
```

Comment:
This test makes it sound like `config.sequence` and `config.kmsModule` are involved in the index, but they aren't.